src/Html2Text.php

```diff
@@ -66,6 +66,7 @@ class Html2Text
         '/(<table[^>]*>|<\/table>)/i',                    // <table> and </table>
         '/(<tr[^>]*>|<\/tr>)/i',                          // <tr> and </tr>
         '/<td[^>]*>(.*?)<\/td>/i',                        // <td> and </td>
+        '/<img(?:.*?)alt=("|\')(.*?)("|\')(?:.*?)>/i',    // img alt text
```

Comment:
It looks like this regex could consume something like `<img><p alt="blah">` You probably want something like `'/<img[^>]*alt=('[^']'|"[^"]")[^>]*>/i'`